lib/Service/ColumnService.php

```diff
@@ -519,4 +523,31 @@ public function getColumnsCount(int $tableId): int {
 			throw new PermissionError('no read access for counting to table id = '.$tableId);
 		}
 	}
+
+	/**
+	 * add some basic values related to this column in context
+	 *
+	 * $userId can be set or ''
+	 * @param Column $column
+	 *
+	 * @return Column
+	 */
+	private function enhanceColumn(Column $column): Column {
+		// add created by display name for UI usage
+		$column->setCreatedByDisplayName($this->userHelper->getUserDisplayName($column->getCreatedBy()));
+		$column->setLastEditByDisplayName($this->userHelper->getUserDisplayName($column->getLastEditBy()));
+		return $column;
+	}
+
+	private function enhanceColumns(?array $columns): array {
+		if ($columns === null) {
+			return [];
+		}
+
+		$out = [];
+		foreach ($columns as $column) {
+			$out[] = $this->enhanceColumn($column);
+		}
+		return $out;
```

Comment:
you should not need `$out` here, as objects are always passed by reference. You can return `$columns`.